build.cake

```diff
@@ -200,14 +206,13 @@ Task("BuildReactiveUI")
 {
     Action<string> build = (solution) =>
     {
-        Information("Building {0}", solution);
-
-        FilePath msBuildPath = VSWhereLatest().CombineWithFilePath("./MSBuild/15.0/Bin/MSBuild.exe");
+        Information("Building {0} using {1}", solution, msBuildPath);
 
         MSBuild(solution, new MSBuildSettings() {
-                ToolPath= msBuildPath
+                ToolPath = msBuildPath,
+                ArgumentCustomization = args => args.Append("/bl:reactiveui.binlog")
             }
-            .WithTarget("restore;pack")
+            .WithTarget("restore;build;pack")
             .WithProperty("PackageOutputPath",  MakeAbsolute(Directory(artifactDirectory)).ToString())
```

Comment:
Had to add `build` here to generate our unit test assemblies (`ReactiveUI.Tests/bin/**/`). Before VS2017 15.3 they were generated automatically via the pack step.